projects/admin/src/app/record/detail-view/location-detail-view/location-detail-view.component.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2019 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { Observable } from 'rxjs';
+
+@Component({
+  selector: 'admin-location-detail-view',
+  templateUrl: './location-detail-view.component.html'
+})
+export class LocationDetailViewComponent implements OnInit, OnDestroy {
+
+  /** Observable resolving record data */
+  record$: Observable<any>;
+
+  /** Resource type */
+  type: string;
+
+  /** Record */
+  record: any;
+
+  /** The observer to the record observable */
+  private recordObs = null;
+
+  /** Constructor */
+  constructor() { }
+
+  /** On init hook */
+  ngOnInit() {
+    this.recordObs = this.record$.subscribe(record => {
+      this.record = record;
+    });
+  }
+
+  /** On destroy hook */
+  ngOnDestroy() { this.recordObs.unsubscribe(); }
```

Comment:
not necessary because the observable emits only one value.

Reply:
Done